### DISCUSSION TITLE
Cannot manage the System > Talkgroups while SDRTrunk is running

### DISCUSSION THREAD

**Discussion by noamlivne** [chuot/rdio-scanner#154] [General]
Cannot manage the System > Talkgroups while SDRTrunk is running.
The talkgroups close immediately.
Windows10 Version 6.2.2

**Comment by chuot** [comment]
That's because the admin dashboard is reloaded everytime its config changes by the auto-populate. Note that the admin dashboard automatically reload its config only if the form hasn't changed (pristine). Try disabling the auto-populate to see if it still closes the talkgroups.

**Reply by noamlivne** [reply]
Thank you very much for the explanation. Makes sense.
